server/src/termdb.server.init.js

```diff
@@ -483,9 +483,18 @@ export function server_init_db_queries(ds) {
 		// value: array of chart types allowed by term types
 		for (const r of ds.cohort.termdb.termtypeByCohort) {
 			if (!r.type) continue // skip ungraphable parent terms
+			//add profile chart types to profile subcohorts
 
 			if (!(r.cohort in supportedChartTypes)) {
 				supportedChartTypes[r.cohort] = new Set(['regression', 'summary', 'facet'])
+				if (ds.label == 'profile')
```

Comment:
should not hardcode a particular `ds.label`. Instead, follow the pattern for `matrix` and other charts being added if they exist in the `allowedChartTypes`
```js
if (ds.cohort.allowedChartTypes.includes(''profileRadar''))  supportedChartTypes[r.cohort].add('profileRadar)
if (ds.cohort.allowedChartTypes.includes(''profileRadarFacility''))  supportedChartTypes[r.cohort].add('profileRadarFacility') // and so on
```